configure-linux.sh

```diff
@@ -30,6 +30,10 @@ function HELP {
    echo ""
    echo "--set-direction-enabled  --Set direction for capturing incoming or outgoing packets (supported on libpcap>=0.9.1)"
    echo ""
+   echo "--asan                   --Build PcapPlusPlus with Address Sanitizer enabled"
+   echo ""
+   echo "--fuzzing                --Build PcapPlusPlus with Clang enabling Coverage for fuzzing and compile existing fuzzers"
```

Comment:
I'm guessing many people don't know what is fuzzing. I'd suggest to add a link that gives more information.

Also, it'd be helpful to explain the difference between this flag and the `--asan` flag